examples/with-fleur/pages/ssg.tsx

```diff
@@ -0,0 +1,24 @@
+import { FleurishNextPage } from '@fleur/next'
+import { Page } from '../components/page'
+import { TimerOps } from '../domains/timer'
+import { getStaticPropsWithFleur } from '../lib/fleur'
+
+interface Props {}
+
+export const SSG: FleurishNextPage<Props> = () => {
+  return <Page />
+}
+
+export default SSG
+
+export const getStaticProps = getStaticPropsWithFleur(async (context) => {
```

Comment:
I put in this kind of wrapper because we don't want to make the user think about the serialize process, but do you have any concerns about the Next.js policy?